setup.py

```diff
@@ -65,42 +65,185 @@
 ]
 
 install_requires = [
```

Comment:
It is better to create a new variable `install_requires_old` with the old values, a new variable `install_requires_pinned` and use one later `install_requires=install_requires_pinned` down in the `setup.py`.

We could also create a new target `production` containing pinned versions, and use `pip install [production]`.